README.md

```diff
@@ -218,7 +218,7 @@ If you run into any issues, [try the troubleshooting steps here](https://modelco
 Use with [Goose](https://block.github.io/goose/), Block's open-source AI agent:
 ```bash
 # Start Goose with the MCP server
-goose session --with-extension "uvx mcp-panther"
+goose session --with-extension "uvx mcp-panther --compat-mode"
```

Comment:
Let's add a sentence or two about --compat-mode in this section so folks know why it exists

Reply:
will do